kurbo/src/common.rs

```diff
@@ -31,8 +31,9 @@ macro_rules! define_float_funcs {
         /// For documentation see the respective functions in the std library.
         #[cfg(not(feature = "std"))]
         pub trait FloatFuncs : Sized + sealed::FloatFuncsSealed {
-            /// Special implementation for signum, because libm doesn't have it.
+            /// Special implementations, because libm doesn't have them.
```

Comment:
Just duplicate this comment for each. I'd prefer if we linked to the docs for these from the std library directly. Also, worth keeping the link to the core floats issue along with the rem_euclid comment.